jax/experimental/host_callback.py

```diff
@@ -1476,11 +1476,9 @@ def _rewrite_eqn(eqn: core.JaxprEqn, eqns: List[core.JaxprEqn],
   """
```

Comment:
This whole code ought to disappear once we have support for effects in Jaxpr.

Reply:
Yes, we should be able to rewrite HCB once we can lower effects away with tokens.

Reply:
Yes, that's the goal